app/move/controllers/assign/confirmation.js

```diff
@@ -0,0 +1,25 @@
+const allocationService = require('../../../../common/services/allocation')
+
+const PersonAssignBase = require('./base')
+
+class ConfirmationController extends PersonAssignBase {
+  middlewareLocals() {
+    super.middlewareLocals()
+    this.use(this.setAssignNext)
+  }
+
+  async setAssignNext(req, res, next) {
+    const allocationId = res.locals.move.allocation.id
+
+    const allocation = await allocationService.getById(allocationId)
+    const unfilledMove =
+      allocation.moves.filter(
+        move => !move.person || move.person === null
+      )[0] || {}
+
+    res.locals.assignNext = unfilledMove.id
```

Comment:
What if we use [`unassignedMoveId`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/608bcfb1f1223fd1da25cadc21b8a1da3694c9d3/app/allocation/views/view.njk#L97) which is the same variable name we use in the allocation section of the app to refer to the same thing. Be nice to make it clear it's referring to the same scenario.

Reply:
👍